app.py

```diff
@@ -0,0 +1,88 @@
+import os
+from flask import Flask, render_template, request
+from flask import make_response, jsonify, session
```

Comment:
Reduce these two lines to one line

Reply:
I have make the changes on both files. Thanks a lot for your help